### DISCUSSION TITLE
Incompatibly with newer Pi 3A+

### DISCUSSION THREAD

**Discussion by varmfskii** [frntc/Sidekick64#94] [Q&A] [answered]
I have a sidekick64 and a RAD. If I use a Pi 3B+ or an older Pi 3A+ with them, the work, but if I try to use a newer Pi 3A+ with either of them, the Pi will not boot. it shows 4 slow and 7 fast flashes on the green LED. I get the same results from two separate newer Pi 3A+ that I have tried. Has anyone been able to use the 2022 revision of the Pi 3A+ with a Sidekick64?

**Comment by frntc** [comment, marked as answer]
To also have the information here: updating the "Raspberry Pi Firmware" on the SD-card solves the problem (thanks for testing!).

See also https://github.com/frntc/RAD/discussions/31#discussioncomment-7471861
